lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -21,41 +21,125 @@
 class WP_Font_Collection {
 
 	/**
-	 * Font collection configuration.
+	 * The unique slug for the font collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var string
+	 */
+	private $slug;
+
+	/**
+	 * The name of the font collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var string
+	 */
+	private $name;
+
+	/**
+	 * Description of the font collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var string
+	 */
+	private $description;
+
+	/**
+	 * Source of the font collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var string
+	 */
+	private $src;
+
+	/**
+	 * Array of font families in the collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var array
+	 */
+	private $font_families;
+
+	/**
+	 * Categories associated with the font collection.
 	 *
 	 * @since 6.5.0
 	 *
 	 * @var array
 	 */
-	private $config;
+	private $categories;
+
 
 	/**
 	 * WP_Font_Collection constructor.
 	 *
 	 * @since 6.5.0
 	 *
 	 * @param array $config Font collection config options.
-	 *  See {@see wp_register_font_collection()} for the supported fields.
-	 * @throws Exception If the required parameters are missing.
+	 *  {
```

Comment:
Nitpick: `{` is supposed to go at the end of the line above.

Reply:
done